packages/nuxt/layouts/default.vue

```diff
@@ -16,7 +16,7 @@
         </SfButton>
       </template>
     </SfTopBar>
-    <SfHeader class="sf-header--multiline" title="Storefront UI" logo="/assets/logo.svg" cartItemsQty="1" is-sticky>
+    <SfHeader class="sf-header--multiline" title="Storefront UI" logo="/assets/logo.svg" cartItemsQty="1">
```

Comment:
Could you explain why there is a change like that? Was it breaking the E2E test?